controllers/administration.js

```diff
@@ -1094,6 +1109,8 @@ router.all(
 					pagination,
 					filterSettings: JSON.stringify(userFilterSettings('lastName', true)),
 					schoolUsesLdap: res.locals.currentSchoolData.ldapSchoolIdentifier,
+					schoolCurrentYear: res.locals.currentSchoolData.currentYear,
+					years,
```

Comment:
@CordlessWool we have added a years property to school model, that replaces the direct call to years service. see https://github.com/schul-cloud/schulcloud-server/blob/develop/src/services/school/hooks/index.js#L47 for more details

Reply:
the final structure is there: https://github.com/schul-cloud/schulcloud-server/blob/develop/src/services/school/logic/year.js#L133